src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -178,13 +178,7 @@ export const createTransaction =
       }
 
       const tx = isExecution ? getExecutionTransaction(txArgs) : getApprovalTransaction(safeInstance, safeTxHash)
-      const sendParams: PayableTx = {
-        from,
-        value: 0,
-        gas: ethParameters?.ethGasLimit,
-        [getGasParam()]: ethParameters?.ethGasPriceInGWei,
-        nonce: ethParameters?.ethNonce,
-      }
+      const sendParams = createSendParams(from, ethParameters || {})
```

Comment:
This is great! Could you not just default the second value to an empty object in the function itself though?